coalib/core/Bear.py

```diff
@@ -130,6 +132,21 @@ class Bear:
     CAN_FIX = set()
     ASCIINEMA_URL = ''
     BEAR_DEPS = set()
+    """
+        Attributes for a new bear.
+
+        :param section:
+            The section object where bear settings are contained.
+        :param file_dict:
+            The file-dictionary containing a mapping of filenames to the
+            according file contents.
+        :raises RuntimeError:
+            Raised when bear requirements are not fulfilled.
+    """
```

Comment:
Hm I want to have this documentation in the constructor somehow... or somewhere available where everyone would look instantly. Is there some possibility? @Siddhartha1234